dist/cms.js

```diff
@@ -4,6 +4,37 @@ function usesBlockTheme() {
   return !!document.querySelector('div.wp-site-blocks');
 }
 
+/**
+ * Detects the WordPress parent and child theme slugs.
+ *
+ * See https://core.trac.wordpress.org/changeset/59698.
+ *
+ * @returns {object} Object with fields `theme` and `parent_theme`.
+ */
+function getWordPressTheme() {
+  const theme = {
+    theme: '',
+    child_theme: '',
```

Comment:
This works, though related to https://github.com/HTTPArchive/custom-metrics/pull/157#discussion_r1931143821 it might still be good to differentiate between "unknown" and "none". I'd say if `wp-theme-...` is present and `wp-child-theme-...` is not present, we should set `child_theme` to `''` (none).

But if neither of the two are present, we for sure don't know (because every site _must_ have at least a theme). So then using another value for both (e.g. `null`) would be good.

Reply:
So null if both are absent? Works for me.

Reply:
Exactly.

Reply:
Maybe also `theme: null` if for some reason a `wp-child-theme-...` class is present, but `wp-theme-...` isn't. That would be invalid and is probably an edge-case, so might also fine to ignore it.